src/gas-hub.js

```diff
@@ -438,93 +457,101 @@ function updateGist() {
+        } else {
+          showAlert('This repository is empty, first create a new branch', LEVEL_WARN);
+        }
+      } else if ($.inArray(branch, branches.map(branch => branch.name)) < 0) {
+        branch = ($.inArray("master", branches.map(branch => branch.name)) >= 0) ? 'master' : branches[0].name;
       }
-    } else if ($.inArray(branch, branches.map(branch => branch.name)) < 0) {
-      branch = ($.inArray("master", branches.map(branch => branch.name)) >= 0) ? 'master' : branches[0].name;
-    }
-    $('#scm-bind-branch').text(`Branch: ${branch}`);
-    //update context and storage
-    context.branch = branch;
-    Object.assign(context.bindBranch, { [context.id] : branch });
-    chrome.storage.sync.set({ bindBranch: context.bindBranch });
-    return branch;
-  });
+      $('#scm-bind-branch').text(`Branch: ${branch}`);
+      //update context and storage
+      context.branch = branch;
+      Object.assign(context.bindBranch, {
+        [context.id]: branch
+      });
+      chrome.storage.sync.set({
+        bindBranch: context.bindBranch
+      });
+      return branch;
+    });
 }
 
 function handleRepoCreated(repo) {
   return scm.getRepos()
-  .then(updateRepo)
-  .then(updateBranch)
-  .then(() => {
-    $('#new-repo-name').val('');
-    $('#new-repo-desc').val('');
-    $('#new-repo-type').val('public');
-    showAlert(`Successfully create new repository ${repo}`);
-  })
-  .catch(() => {
-    throw new Error('Repository created, but failed to show the new repository.');
-  });
+    .then(updateRepo)
+    .then(updateBranch)
+    .then(() => {
+      $('#new-repo-name').val('');
+      $('#new-repo-desc').val('');
+      $('#new-repo-type').val('public');
+      showAlert(`Successfully create new repository ${repo}`);
+    })
+    .catch(() => {
+      throw new Error('Repository created, but failed to show the new repository.');
+    });
 }
 
 function handleBranchCreated(branch) {
   return updateBranch()
-  .then(() => {
-    $('#new-branch-name').val('');
-    showAlert(`Successfully create new branch: ${branch}`);
-  })
-  .catch(() => {
-    throw new Error('Branch created, but failed to show the new branch.');
-  });
+    .then(() => {
+      $('#new-branch-name').val('');
+      showAlert(`Successfully create new branch: ${branch}`);
+    })
+    .catch(() => {
+      throw new Error('Branch created, but failed to show the new branch.');
+    });
 }
 
 function handleGistCreated() {
   return updateGist()
-  .then(() => {
-    $('#new-gist-name').val('');
-    $('#new-gist-public').val('public');
-    showAlert(`Successfully create new gist.`);
-  })
-  .catch(err => {
-    throw new Error('Gist created, but failed to show the new gist.');
-  });
+    .then(() => {
+      $('#new-gist-name').val('');
+      $('#new-gist-public').val('public');
+      showAlert(`Successfully create new gist.`);
+    })
+    .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/gas-hub.js#L552"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>